stories/index.stories.js

```diff
@@ -6,6 +6,8 @@ import { linkTo } from '@storybook/addon-links';
 
```

Comment:
Please add more stories for different states of the login component, based on props.